src/_scss/pages/search/_tooltips.scss

```diff
@@ -53,6 +53,18 @@ $letter-color: #323A45;
                 padding-left: rem(40);
                 padding-top: 0;
             }
+            
+            .advanced-search__glossary-term {
+                font-weight: $font-semibold;
+
+                .usda-glossary-link{
```

Comment:
this can be removed since it's no longer a link

Reply:
removed